### ISSUE TITLE
Changing URL and registering with LetsEncrypt?

### ISSUE THREAD

**Issue by DataSpotting** [damalis/full-stack-nginx-wordpress-for-everyone-with-docker-compose#4]
I've looked through all the files on my set-up to see what I would need to change to change the domain url and retrigger letsencrypt for the new domain but I've not been successful. What do I have to do to have the same set-up with the new url. I'm super new to docker in general so this may just be a knowledge gap, but I would appreciate some help. Thank you 😅

**Comment by damalis** [comment]
You could try to remove (all set-up) with `docker compose down -v`  then to exec again "install.sh" for new domain.

**Comment by datastraine** [comment]
Thanks for the advice. I managed to contact my friend who knows more about this and networking than I do and he was able to quickly walk me through it as well. For others who may have the same issue 

* Change the URL in wordpress dashboard or phpadmin interface (I did wordpress dashboard)
* Take down the container with `docker compose down`
* Change the environmental file and replace the domain name with the new one (and new e-mail if applicable)
*  cd ./phpmyadmin/apache2/sites-available/ and change the default-ssl.conf to have the new domain name and aliases it uses (and e-mail if applicable)
* cd back to the root docker dir and run `docker-compose up -d --no-deps --build`
* If that step doesn't work, restart the server and then run `docker-compose up -d --no-deps --build` (I did this step, but I feel like it shouldn't be needed but in case it is 🤷‍♂️ )

Hopefully, this helps others!

**Comment by DataSpotting** [comment]
Thanks for the advice. I managed to contact my friend who knows more about this and networking than I do and he was able to quickly walk me through it as well. For others who may have the same issue 

* Change the URL in wordpress dashboard or phpadmin interface (I did wordpress dashboard)
* Take down the container with `docker compose down`
* Change the environmental file and replace the domain name with the new one (and new e-mail if applicable)
*  cd ./phpmyadmin/apache2/sites-available/ and change the default-ssl.conf to have the new domain name and aliases it uses (and e-mail if applicable)
* cd back to the root docker dir and run `docker-compose up -d --no-deps --build`
* If that step doesn't work, restart the server and then run `docker-compose up -d --no-deps --build` (I did this step, but I feel like it shouldn't be needed but in case it is 🤷‍♂️ )

Hopefully, this helps others!
